src/safeposix/syscalls/fs_calls.rs

```diff
@@ -1242,18 +1290,55 @@ impl Cage {
     }
 
     //------------------------------------FSTAT SYSCALL------------------------------------
+    /// ### Description
```

Comment:
A line stating that the function has the same functionality as fstat - the only difference is the file indicator (i.e path if stat and fd ion fstat) would be good I think

Reply:
yeah I had considered this exact thing, but I thought being explicit about everything instead of a slightly vague statement like "same as stat except in xyz" wouldn't really be helpful for end users. Plus this would then have to be replicated everywhere there is commonality